### PR TITLE
Multi-send tx service

### PR DESCRIPTION
**Pull request by nerfZael**
**Changes:**
- Sending transactions to tx service no longer calculates gas (avoids simulation errors with dependent txs)
- Implemented and using sending multi send tx for tx batch

### CODE REVIEW

File: autotx/wallets/safe_smart_wallet.py [hunk 1]
```diff
@@ -17,4 +17,4 @@ def on_transactions_prepared(self, txs: list[models.Transaction]) -> None:
         pass
 
     def on_transactions_ready(self, txs: list[models.Transaction]) -> bool | str:
-        return self.manager.send_tx_batch(txs, not self.auto_submit_tx)
+        return self.manager.send_multisend_tx_batch(txs, not self.auto_submit_tx)
```
Comment (cbrzn) [inline autotx/wallets/safe_smart_wallet.py:20]: do we want to send it as multisend even when there's just one transaction in the list?

**Review by cbrzn — APPROVED** [review]

File: autotx/wallets/safe_smart_wallet.py (hunk 1 above)
Comment (nerfZael) [reply autotx/wallets/safe_smart_wallet.py:20]: I changed it to not be multi send in case of a single tx

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token

**Review by dOrgJelli — COMMENTED** [review]
LGTM!

**Pull request merged**